AggregateMetrics/AggregateMetrics/AzureWebApp/AzureWebApp.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.Two;
+using System;
+using Microsoft.ApplicationInsights.DataContracts;
+using System.Net.Http;
+using System.Runtime.Caching; 
+
+namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+{
+
+    class PerformanceCounterGauge : ICounterValue
+
+    {
+        public string GetJson()
+        {
+            HttpClient client = new HttpClient();
+            var t = client.GetStringAsync("http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_APP/");
```

Comment:
What is "var t"?
I'm guessing this should be something like "Task<string> evnironmentVariables" or "...websiteCounters" based on the URL but I really can't tell from reading the code.